CRISPResso2/args.json

```diff
@@ -255,6 +255,13 @@
             "default": "None",
             "tools": ["Core", "Batch", "Pooled", "WGS"]
```

Comment:
@kclem A few questions about this implementation. 

- In `get_n_aligned_bam` https://github.com/edilytics/CRISPResso2/blob/cole/parameterize-bam-flags/CRISPResso2/CRISPRessoPooledCORE.py#L158 and `get_n_aligned_bam_region` https://github.com/edilytics/CRISPResso2/blob/cole/parameterize-bam-flags/CRISPResso2/CRISPRessoPooledCORE.py#L162 we filter for unmapped (and I'm not sure what the `0x9` flag is). Do you think we should also apply the custom exclude flags to these functions?
- In CRISPRessoCORE, we don't filter the unmapped reads <https://github.com/edilytics/CRISPResso2/blob/cole/parameterize-bam-flags/CRISPResso2/CRISPRessoCORE.py#L822>. If we add the `args.samtools_exclude_flags` to this call, it will change the default behavior because the unmapped reads will get filtered. Do you think this is desired?